### PR TITLE
[ES] Create 13-2.md

### PR DESCRIPTION
**Pull request by eugeniaft**
Algunas traducciones que no estaba segura:

k-hop : traduje como k-saltos
Vanilla Spectral GCN : GCN Spectral simple
Template Matching: Coincidencia de Modelos (lo vi de esa forma en el archivo 13-1.md)
Vanilla Spatial GCNs: GCN Espacial simple
Graph Attention Networks:  Redes de grafos de atención
Gated Graph ConvNets: ConvNets de grafos cerrados
edge gating mechanism: mecanismo de cerrado de borde (google translate lo traducía como mencanimso de compuerta de borde pero me sonaba raro)

### CODE REVIEW

File: docs/es/week13/13-2.md [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+Esta técnica fue la primera técnica espectral utilizada para ConvNets, pero tiene algunas limitaciones:
+* No hay garantía de localización espacial de filtros
+* Es preciso aprender $O(n)$ parámetros por capa ($\hat{w}(\lambda_1)$a$\hat{w}(\lambda_n)$)
+* La tasa de aprendizaje es $O(n^2$) porque $\boldsymbol{\phi}$ es una matriz densa
+
+
+<!-- ### SplineGCNs
+SplineGCNs involve computing smooth spectral filters to get localized spatial filters. The connection between smoothness in frequency domain and localization in space is based on Parseval's Identity (also Heisenberg uncertainty principle): smaller derivative of spectral filter (smoother function) $\Leftrightarrow$ smaller variance of spatial filter (localization).
+How do we get a smooth spectral filter? We decompose the spectral filter to be a linear combination of $K$ smooth kernels $\boldsymbol{B}$ (splines) so that $\hat{w}^l(\Lambda)=diag(\boldsymbol{B}w^l)$. The activation equation the is as the following.
+ -->
```
Comment (GastonMazzei) [inline docs/es/week13/13-2.md:59]:
```suggestion
$$ h^{l+1}=\eta\bigg(\boldsymbol{\phi} \Big(\text{diag}(\boldsymbol{B}w^l)\Big)\boldsymbol{\phi^\top} h^l\bigg) $$
 -->
```
just adding the original last-equation which was missing

File: docs/es/week13/13-2.md [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+
+$$
+h^{l+1}=\eta\bigg(\sum^{K-1}_{k=0}w_kX_k\bigg)=\eta\Big((w^l)^\top \bar{X}\Big)
+$$
+
+<!-- Note: Since no Laplacian eigen-decomposition is used, all operations are in the  spatial (not spectral) domain, so calling them Spectral GCNs may be misguided. Further, another drawback of LapGCNs is that convolutional layers involve sparse linear operations, which GPU's are not fully optimized for.
+ -->
+<!-- We now have resolved the 3 limitations of Vanilla GCNs through localized filters (in $K$-hop support), $O(1)$ parameters per layer and $O(n)$ learning complexity. However, the limitation of LapGCNs is that monomial basis ($\Delta^0,\Delta^1,\ldots$) used is unstable for optimization because it is not orthogonal (changing one coefficient changes the function approximation).
+ -->
+Nota: Dado que no se utiliza la descomposición de Laplace en base a raíces características, todas las operaciones están en el dominio espacial (no espectral), por lo que llamarlas GCN espectrales puede estar equivocado. Además, otro inconveniente de las LapGCN es que las capas convolucionales implican operaciones lineales dispersas, para las que los GPU no están completamente optimizados.
```
Comment (GastonMazzei) [inline docs/es/week13/13-2.md:118]: suggestion: "Nota: Dado que no se descompone espectralmente el Laplaciano,..."

File: docs/es/week13/13-2.md [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+<!-- where, $h_{i}^{l+1}$ has the dimensions of $d \times 1$
+
+The vectorial representation is responsible for handling the absence of node ordering, which is invariant of node re-parametrisation. That is, adding on the previous example, if the node has an in 6 and is changed to 122, this won't change anything in the computation of the activation function of the next layer $h^{l+1}$.
+
+We can also deal with neighbourhood of different sizes. That is we can have a neighbourhood of 4 nodes or 10 nodes, it wouldn't change anything. -->
+donde, $h_{i}^{l+1}$ tiene las dimensiones de $d \times 1$
+
+La representación vectorial es responsable de manejar la ausencia de ordenamiento de nodos, que es invariante a la re-parametrización de nodos. Es decir, agregando al ejemplo anterior, si el nodo tiene un índice 6 y se cambia a 122, esto no cambiará nada en el cálculo de la función de activación de la siguiente capa $ h ^ {l + 1} $.
+
+También podemos ocuparnos de vecindarios de diferentes tamaños. Es decir, podemos tener un vecindario de 4 o 10 nodos, no cambiaría nada.
```
Comment (GastonMazzei) [inline docs/es/week13/13-2.md:266]:
```suggestion
También podemos ocuparnos de vecindarios de diferentes tamaños. Es decir; podemos tener un vecindario de 4 o 10 nodos y no cambiaría nada.
```

File: docs/es/week13/13-2.md [hunk 4]
```diff
@@ -0,0 +1,569 @@
...
+
+<!-- We are given the local reception field by design, that is, with Graph Neural Networks we only have to consider the neighbours.
+
+We have weight sharing, that is, we the same $\boldsymbol{W}^{l}$ matrix for all features no matter the position of the graph, which is a Convolution property.
+
+This formulation is also independent of the graph size, since all operations are done locally.
+
+Since it is an isotropic model, the neighbours will have the same $\boldsymbol{W}^{l}$ matrix.
+ -->
+Se nos da el campo de recepción local por diseño, es decir, con Redes de Grafo Neuronales solo tenemos que considerar los vecinos.
```
Comment (GastonMazzei) [inline docs/es/week13/13-2.md:276]:
```suggestion
El campo de recepción local está dado por diseño. Es decir, con Redes de Grafo Neuronales solo tenemos que considerar los vecinos.
```

File: docs/es/week13/13-2.md [hunk 5]
```diff
@@ -0,0 +1,569 @@
...
+
+Tenemos peso compartido, es decir, tenemos la misma matriz $\boldsymbol{W}^{l}$ para todas las características sin importar la posición del grafo, que es una propiedad de convolución.
```
Comment (GastonMazzei) [inline docs/es/week13/13-2.md:278]:
```suggestion
Tenemos peso compartido. Es decir, tenemos la misma matriz $\boldsymbol{W}^{l}$ para todas las características sin importar la posición del grafo, lo cual es una propiedad de la convolución.
```

File: docs/es/week13/13-2.md [hunk 6]
```diff
@@ -0,0 +1,569 @@
...
+ -->
+## [GCN anisotrópicas] (https://www.youtube.com/watch?v=Iiv9R6BjxHM&list=PLLHTzKZzVU9eaEyErdV26ikyolxOsz6mq&index=24&t=5586s)
+
+Las CNN estándar tienen la capacidad de producir filtros anisotrópicos, algunos que favorecen ciertas direcciones. Esto se debe a que la estructura direccional se basa en arriba, abajo, izquierda y derecha. Sin embargo, las GCN descritas anteriormente no tienen noción de dirección y, por lo tanto, solo pueden producir filtros isotrópicos. La anisotropía se puede introducir de forma natural, con características de borde. Por ejemplo, las moléculas pueden tener enlaces simples, dobles y triples. Gráficamente, se introduce ponderando diferentes vecinos de forma diferente.
```
Comment (GastonMazzei) [inline docs/es/week13/13-2.md:383]:
```suggestion
Las CNN estándar tienen la capacidad de producir filtros anisotrópicos (que favorecen ciertas direcciones). Esto se debe a que la estructura direccional se basa en arriba, abajo, izquierda y derecha. Sin embargo, las GCN descritas anteriormente no tienen noción de dirección y, por lo tanto, solo pueden producir filtros isotrópicos. La anisotropía puede ser introducida de forma natural con características de borde. Por ejemplo, las moléculas pueden tener enlaces simples, dobles y triples. Gráficamente, se introduce ponderando distintos vecinos de forma diferente.
```

File: docs/es/week13/13-2.md [hunk 7]
```diff
@@ -0,0 +1,569 @@
...
+<center>
+<img src="{{site.baseurl}}/images/week13/13-2/Figure12.png" /><br>
+<b>Figure 12:</b> Graph Classification task
+</center>
+
+<center>
+<img src="{{site.baseurl}}/images/week13/13-2/Figure13.png" /><br>
+<b>Figure 13:</b> Performance of various GCNs on Graph Classification task
+</center> -->
+Observamos que en la mayoría de los casos las GCN anisotrópicos funcionan mejor en comparación con las GCN isotrópicos porque usamos propiedades direccionales.
```
Comment (GastonMazzei) [inline docs/es/week13/13-2.md:508]:
```suggestion
Observamos que en la mayoría de los casos las GCN anisotrópicos funcionan mejor en comparación con las GCN isotrópicas porque usamos propiedades direccionales.
```

File: docs/es/week13/13-2.md [hunk 8]
```diff
@@ -0,0 +1,569 @@
...
+There is linear complexity for sparse graphs, and GPU implementation, although the latter is not yet optimized for sparse matrix multiplication. The applications are abound as shown below.
+
+<center>
+<img src="{{site.baseurl}}/images/week13/13-2/Figure16.png" /><br>
+<b>Figure 16:</b> Applications
+</center>
+ -->
+## Conclusión
+
+Las GCN generalizan las CNN a datos en grafo. El operador de convolución necesitaba ser rediseñado para grafos. Hacer esto para la coincidencia de modelos dio lugar a GCN espaciales, y para la convolución espectral condujo a GCN espectrales.
```
Comment (GastonMazzei) [inline docs/es/week13/13-2.md:562]:
```suggestion
Las GCN generalizan las CNN a datos en grafos. El operador de convolución necesitaba ser rediseñado para grafos. Hacer esto para la coincidencia de modelos dio lugar a GCN espaciales, y para la convolución espectral condujo a GCN espectrales.
```

**Review by GastonMazzei — COMMENTED** [review]
It was excellent! added mostly suggestions and marked some minor errors.

**Comment by JonathanSum** [comment]
Don't forget to put [ES] at the starting of the title.

**Comment by eugeniaft** [comment]
> It was excellent! added mostly suggestions and marked some minor errors.

Thanks for reviewing! I added your suggestions!

**Review by GastonMazzei — APPROVED** [review]
I have found no more errors.

Good translation. Very easy to read. Thank you for your time!

**Review by Atcold — APPROVED** [review]
Thanks! I've updated `docs/_config.yml` for you.

**Pull request merged**